pkgs/development/python-modules/google-crc32c/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, cffi }:
+
+buildPythonPackage rec {
+  pname = "google-crc32c";
+  version = "1.0.0";
+
```

Comment:
Probably need to add `disabled = !isPy3k;` here so it doesn't show up as a "failing build" forever.